components/comparision/ComparisonOptionCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="comparison-option-select__card">
+    <div class="comparison__logo-wrapper d-flex justify-content-center">
+      <img
+        v-if="modelSelectedOption"
+        :src="formBuildersLogoSrc[modelSelectedOption]"
+      />
+      <nuxt-img v-else src="/form-building-platforms/plus.svg" />
+    </div>
+    <div class="comparison__dropdown-wrapper">
+      <select
+        class="form-select"
+        v-model="modelSelectedOption"
+        @change="emitChange"
+      >
+        <option disabled value="">Select a tool</option>
+        <option v-for="option in options" :key="option" :value="option">
+          {{ option }}
+        </option>
+      </select>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    cardNumber: {
+      type: Number,
+      required: true,
+    },
+    options: {
+      type: Array,
+      required: true,
+    },
+    selectedOption: {
+      type: String,
+      default: '',
+    },
+    formBuildersLogoSrc: {
+      type: Object,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      modelSelectedOption: this.selectedOption,
+    }
+  },
+  methods: {
+    emitChange() {
+      this.$emit('change', this.modelSelectedOption, this.cardNumber)
+    },
+  },
+}
+</script>
+
+<style scoped>
+.comparison-option-select__card {
+  border-radius: 4px;
+  border: 1px solid #eaecf0;
```

Comment:
Don't we have variables for the border-color?

Reply:
This is a website so, the card color may vary at different places.

Reply:
and this is a newly made card, we don't have this color in our color variable list.